samples/bookinfo/routing-rules/route-rule-all-v1.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage-route
+spec:
+  hosts:
+  - productpage
+  gateways:
+  - bookinfo-gateway
+  - mesh #TODO remove this if not needed
+  http:
+  - route:
+    - destination:
+        name: productpage
+        subset: v1
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews-route
+spec:
+  hosts:
+  - reviews
+  http:
+  - route:
+    - destination:
+        name: reviews
+        subset: v1
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings-route
+spec:
+  hosts:
+  - ratings
+  http:
+  - route:
+    - destination:
+        name: ratings
+        subset: v1
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: details-route
+spec:
+  hosts:
+  - details
+  http:
+  - route:
+    - destination:
+        name: details
+        subset: v1
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: productpage-destination
+spec:
+  name: productpage
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+---
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews-destination
+spec:
+  name: reviews
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+  - name: v3
+    labels:
+      version: v3
```

Comment:
isn't this a duplicate of the destination-policy-reviews-... ?

Reply:
It's an example resource to change the loadbalancing policy ... unfortunately you can't currently patch in the changes, so you need to duplicate the subset part and use `istioctl replace ...`.